e2e/tests/contracts.rs

```diff
@@ -1084,16 +1086,21 @@ async fn test_add_custom_assets() -> Result<()> {
 async fn contract_load_error_messages() {
     {
         let binary_path = "sway/contracts/contract_test/out/release/no_file_on_path.bin";
```

Comment:
Can you add some no-op here so that we see the 1.79 improved err in action? 

E.g. `sway/contracts/../../sway/contracts/contract_test...`